src/Utilities/PaymentUtils.res

```diff
@@ -185,7 +186,21 @@ let getDisplayName = (
     })
     ->Belt.Array.get(0)
   switch customNameObj {
-  | Some(val) => val.paymentMethodName === "classic" ? val.aliasName : defaultName
-  | None => defaultName
+  | Some(val) =>
+    val.paymentMethodName === "classic"
+      ? {
+          let id = val.aliasName->Js.String2.split(" ")
+          (
+            val.aliasName,
+            Some(
+              PaymentMethodsRecord.icon(
+                id->Belt.Array.get(0)->Belt.Option.getWithDefault(""),
```

Comment:
Can you do this using switch case
```
switch id->Belt.Array.get(0) {
| Some(iconName) => PaymentMethodsRecord.icon(iconName, ~size=19)
| None => defaultIcon
}
```

Reply:
we dont want this default icon as a fallback here.. so cant really do that